### DISCUSSION TITLE
Black/grey line in spectrogram till the clicked poistion

### DISCUSSION THREAD

**Discussion by puneety95** [katspaugh/wavesurfer.js#3213] [Q&A] [answered]
## Bug description
I am rendering spectrogram instead of waves in the wavesurfer. Sometimes not always there a black/grey line in between till the pointer. Can you please help in removing it.

## Environment
 - Browser: Chrome

## Screenshots
![Screenshot 2023-09-27 at 12 21 25 PM](https://github.com/katspaugh/wavesurfer.js/assets/79069943/24053148-699d-47ad-a47c-0f39d32f264f)

**Comment by katspaugh** [comment]
You're doing something custom and I've no idea what's going on that screenshot.

**Reply by puneety95** [reply]
I am talking about a horizontal grey line in the center till the red vertical line which is basically the pointer where I clicked.

**Reply by katspaugh** [reply, marked as answer]
Try setting `waveColor: transparent, progressColor: transparent` in your wavesurfer options.

**Reply by puneety95** [reply]
Thanks @katspaugh  it worked.
